polybar-scripts/xkcd-notifier/notifier.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+import requests
+from sys import argv
+
+latest_file = "latest"
+prefix = "#"
+read = False
+
+help = """xkcd-notifier: Display the current XKCD comic number, and show if there are any new ones.
+
+Usage:
+	-h, --help	Display this message
```

Comment:
Actually I would not use arguments. I would use directly variables, which can be modified by everyone. 
This makes the script shorter. But that is your decision here.

Reply:
That sounds like a good idea, I've done that